task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +91,8 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    const perimetre = a + b + c;
+    return [a, b, c].filter(a => a < perimetre - a).length === 3 ? true : false;
```

Comment:
Actually, we don't need ternary operator here
just return [a, b, c].filter(a => a < perimetre - a).length === 3
Please, see https://javascript.info/logical-operators

Reply:
useless ternary operator is removed
logical-operators article is read